src/org/opensolaris/opengrok/configuration/Project.java

```diff
@@ -68,6 +68,12 @@
      */
     private boolean navigateWindowEnabled = false;
 
+    /**
+     * This flag sets per-project handling of renamed files.
+     */
+    private enum propState { YES, NO, UNSET };
+    private propState handleRenamedFiles = propState.UNSET;
```

Comment:
Hi. Perhaps `Boolean` (i.e. a nullable object wrapping `boolean` and therefore tri-state) would do here just as well?

Reply:
good idea!